### ISSUE TITLE
[FEATURE REQUEST] More Individual Devices

### ISSUE THREAD

**Issue by haroldboom** [flixlix/power-flow-card-plus#676]
**Is your feature request related to a problem? Please describe.**

I have more than 4 individual devices that I would like to display, I understand that only 4 can fit but most of the time it is only displaying one device and the rest are hidden.

**Describe the solution you'd like**

I would like to have unlimited individual devices and just have a priority of which 4 are displayed. I have my devices hidden on 0 watts, so you could have display priority of the top 4 highest instant power usage or maybe just priority of where they appear in the list.

**Describe alternatives you've considered**
Possibly use a max sensor to combine devices but I would rather have them as individual 

**Additional context**
N/A

**Comment by IIIdefconIII** [comment]
That actually sound nicex it it also possible to add floors?

**Comment by alucryd** [comment]
Displaying the 4 highest power consumption in the list would be cool too.

**Comment by SBergers** [comment]
Yes. When Home Assistant brought in the Untracked Consumption in the default energy dashboard.. I invested in several power meters. I love how this plugin pops the devices that are turned on and running in and out when their turned off. You even built in a treshold for my Oven usecase, which always displays time therefore takes up 1w or so. Love the plugin, would love to have all my devices in there so when Home-energy usage is high, i see directly which device it is.

**Comment by ptrkp** [comment]
Yes, please, remove the 4 limit

**Comment by mazetar** [comment]
I would love a card that let me show all my device groups/devices.
So far I'd love to have 12 devices to handle my entire house.

They way I'm currently attempting to handle it is by cheating with having a group called "Heating" which shows the total for all my heaters.
Then I have another flow card where I tell it that all heaters is my "grid usage" and then I use devices there to seperate into floors.
Something like this:
![image](https://github.com/user-attachments/assets/bdab5f31-9223-447b-8237-6174cffcea0f)

**Comment by IIIdefconIII** [comment]
I just use the card twice


Hartelijke groet,

________________________ _______________________
Thomas van Dalen

E: ***@***.******@***.***>

T: indien nodig

________________________________

**Comment by IIIdefconIII** [comment]
Yeah exactly.likr that's didn tsee the picture on Outlook

**Comment by IIIdefconIII** [comment]
Yeah exactly.likr that's didn tsee the picture on Outlook ![Screenshot_20241026_123037.jpg](https://github.com/user-attachments/assets/161f86a7-06e2-4f2a-9a13-7c5295430e8f)

![Screenshot_20241026_123042.jpg](https://github.com/user-attachments/assets/d190e9cd-f222-40a1-8513-d8a31e89132c)

I did that with my lights

**Comment by IIIdefconIII** [comment]
Samer here! would be lovely if we can add more devices and even group them like click on lights nex t to my setup and open a sub card.

![image](https://github.com/user-attachments/assets/e96579e5-7d3a-4294-9e3f-1a8aa9def342)

**Comment by developersteve** [comment]
X2 on this as well, Im not familiar with the code base (yet) but is there a place where we could start to look at adding more than 4 individual devices?

**Comment by flixlix** [comment]
as of `v0.3.1`, you can define unlimited individuals and show the top 4 consumers!
